app/controllers/auth_controller.py

```
@@ -0,0 +1,12 @@
+from fastapi import Depends, HTTPException
+from app.services.user_service import UserService
+
+
+class AuthController:
+    def __init__(self, user_service: UserService):
+        self.user_service = user_service
+
+    async def login(self, email: str, password: str):
+        user = await self.user_service.get_user(email)
```

Ruff (0.8.2)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)